nav2_util/include/nav2_util/geometry_utils.hpp

```diff
@@ -119,6 +121,44 @@ inline double euclidean_distance(
   return std::hypot(dx, dy);
 }
 
+/**
```

Comment:
All functions need unit testing

Reply:
- [ ] add unit tests for new helper functions